runtime/jvmti/jvmtiStackFrame.c

```diff
@@ -465,7 +465,7 @@ jvmtiPopFrame(jvmtiEnv* env,
 			/* Error if the thread is not suspended and not the current thread. */
 			if ((currentThread != targetThread)
 #if JAVA_SPEC_VERSION >= 21
-			&& (0 == J9OBJECT_U32_LOAD(currentThread, threadObject, vm->isSuspendedInternalOffset))
+			&& (J9_ARE_NO_BITS_SET(J9OBJECT_U64_LOAD(currentThread, threadObject, vm->isSuspendedInternalOffset), 1))
```

Comment:
we should add helper methods for such checks; in the future, we will only need to update the helper method, instead of updating all the instances.